public/pull_request_template

```diff
@@ -0,0 +1,20 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+## Name of Contributor(s)
+
+## Type of change
```

Comment:
This could be a GitHub `label` (easier to search for). So we could remove this section.